### PR TITLE
FLUT-6329- [Feature] UG document for picker

### PR DESCRIPTION
**Pull request by muthulakshmi-123**
## Description ##

I have changed the order of month cell customization and year cell customization in the flutter date picker. The prepared content was reviewed by @laksmipriyamurugesan 

The content review task link is followed by https://syncfusion.atlassian.net/browse/DOC-4172

### CODE REVIEW

File: Flutter/DateRangePicker/customizations.md [hunk 1]
```diff
@@ -66,6 +66,15 @@ Widget build(BuildContext context) {
 
 ![Customizations Date Range Picker](images/customizations/customizations.png)
 
+>**NOTE**
+* Priority order
```
Comment (Nijamudeen-Mohamed) [inline Flutter/DateRangePicker/customizations.md:70]: Cell decoration will be rendered in this order blackout dates, special dates, disable dates, today date, weekend dates,  trailing and leading dates and dates decoration based on the decoration given

File: Flutter/DateRangePicker/customizations.md [hunk 2]
```diff
@@ -169,6 +178,12 @@ Widget build(BuildContext context) {
 
 ![Year cell customization Date Range Picker](images/customizations/yearcell_customization.png)
 
+>**NOTE**
+* Priority order
```
Comment (Nijamudeen-Mohamed) [inline Flutter/DateRangePicker/customizations.md:182]: change like above comment

File: Flutter/DateRangePicker/customizations.md (hunk 1 above)
Comment (muthulakshmi-123) [reply Flutter/DateRangePicker/customizations.md]: I have checked and changed it like this. I have updated in my latest commit.

File: Flutter/DateRangePicker/customizations.md (hunk 2 above)
Comment (muthulakshmi-123) [reply Flutter/DateRangePicker/customizations.md]: I have checked and changed it like above comment. I have updated in my latest commit.

File: Flutter/DateRangePicker/customizations.md [hunk 3]
```diff
@@ -66,6 +66,9 @@ Widget build(BuildContext context) {
 
 ![Customizations Date Range Picker](images/customizations/customizations.png)
 
+>**NOTE**
+* Cell decoration will be rendered in this order blackout dates, special dates, disable dates, today date, weekend dates, trailing and leading dates and dates decoration based on the given decoration.
```
Comment (laksmipriyamurugesan) [inline Flutter/DateRangePicker/customizations.md:70]: Based on the given decoration, cell decoration will be rendered in the following order: blackout dates, special dates, disable dates, today date, weekend dates, trailing and leading dates, and dates decoration.

File: Flutter/DateRangePicker/customizations.md [hunk 4]
```diff
@@ -169,6 +172,9 @@ Widget build(BuildContext context) {
 
 ![Year cell customization Date Range Picker](images/customizations/yearcell_customization.png)
 
+>**NOTE**
+* Cell decoration will be rendered in this order disable dates, today date, trailing and leading dates and dates decoration based on the given decoration.
```
Comment (laksmipriyamurugesan) [inline Flutter/DateRangePicker/customizations.md:176]: Refer the above comment.

File: Flutter/DateRangePicker/customizations.md (hunk 3 above)
Comment (muthulakshmi-123) [reply Flutter/DateRangePicker/customizations.md]: I have changed and updated this in my latest commit

File: Flutter/DateRangePicker/customizations.md (hunk 4 above)
Comment (muthulakshmi-123) [reply Flutter/DateRangePicker/customizations.md]: I have changed and updated this in my latest commit

File: Flutter/DateRangePicker/customizations.md [hunk 5]
```diff
@@ -66,6 +66,9 @@ Widget build(BuildContext context) {
 
 ![Customizations Date Range Picker](images/customizations/customizations.png)
 
+>**NOTE**
+* Based on the given decoration, cell decoration will be rendered in the following order: blackout dates, special dates, disable dates, today date, weekend dates, trailing and leading dates, and dates decoration.
```
Comment (Sasikumar3595) [inline Flutter/DateRangePicker/customizations.md:70]: Leading or trailing dates

Comment (muthulakshmi-123) [reply Flutter/DateRangePicker/customizations.md]: I have checked and updated in my latest commit

**Review by Nijamudeen-Mohamed — APPROVED** [review]
Fine

**Pull request merged**